plugins/arclite/archive.cpp

```diff
@@ -233,6 +233,7 @@ class MyCompressCodecsInfo : public ICompressCodecsInfo, public IHashers, privat
       }
       if (arc_lib.ComHashers) {
         UInt32 numHashers = arc_lib.ComHashers->GetNumHashers();
+        hashers_.reserve(numHashers);
```

Comment:
Thinking about it, all this code is a part of a bigger loop, and calling `reserve` in a loop is a known antipattern: it is too easy to pessimise exponential growth down to linear.

I suggest leaving it as is, it's not like like this place is a bottleneck anyway.